Sources/HAP/Base/Generated.swift

```diff
@@ -96,6 +173,115 @@ public extension ServiceType {
 	static let windowCovering = ServiceType(0x008C)
 }
 
+extension ServiceType: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case let .appleDefined(typeCode):
+            let hex = String(typeCode, radix: 16).uppercased()
```

Comment:
I'd prefer not to convert the value here, and it seems we can easily prevent this. See my original suggestion that just uses the hex values, similar to how we're defining the cases. 
```swift
    public var description: String {
        switch self {
        case let .appleDefined(hex):
            switch hex {
            case 0x0239: return "Accessory Runtime Information"
            case 0x00BB: return "Air Purifier"
            // ...
            default: fatalError("Unknown Apple-defined service type '\(hex)'")
            }
        case let .custom(uuid):
            return "Other \(uuid)"
        }
    }
```

Reply:
No problem, just updated